### PR TITLE
# Change VirtualEnv, fix test cases

### PR DESCRIPTION
**Pull request by takesei**
# WHAT
- Change Virtual Environment tool, from `pipenv` to `poetry`
- Change Build management tool, from setuptools to poetry
- Fix test cases which

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+* @taka-sho
+
+/tagupy @takesei @yo-aka-gene @gk-hazard
```
Comment (takesei) [inline .github/CODEOWNERS:3]: I set the CODEOWNER.
- @taka-sho will be responsible for the whole code
- @yo-aka-gene @gk-hazard  will be responsible for DoE content

When someone tries to edit the code and send the PR, people who are assigned as CODEOWNER need reviewing.

**Review by gk-hazard — APPROVED** [review]
LGTM

**Review by yo-aka-gene — APPROVED** [review]
LGTM

**Comment by takesei** [comment]
@taka-sho Could you please review this PR?

File: docs/conf.py [hunk 2]
```diff
@@ -47,8 +47,8 @@
 
 # General information about the project.
 project = 'TaguPy'
-copyright = "2021, Sei Takeda"
-author = "Sei Takeda"
+copyright = "2021, mogani"
+author = "mogani"
```
Comment (taka-sho) [inline docs/conf.py:51]: 🦀

**Review by taka-sho — APPROVED** [review]
LGTM

**Pull request merged**